README.rst

```diff
@@ -1,11 +1,12 @@
-pep8 - Python style guide checker
+pycodestyle (formerly called pep8) - Python style guide checker
 =================================
 
-pep8 is a tool to check your Python code against some of the style
+pycodestyle is a tool to check your Python code against some of the style
 conventions in `PEP 8`_.
 
 .. _PEP 8: http://www.python.org/dev/peps/pep-0008/
 
+.. note:: This package used to be called :module:`pep8` but was renamed to :module:`pycodestyle` to reduce confusion. Further discussion here: `<https://github.com/PyCQA/pycodestyle/issues/466>`_
```

Comment:
agreed on all points. I got bit by GitHub's web editor UI and didn't realize it wasn't wrapped. 

fixed in latest commit